### PR TITLE
Add YAML anchor usage test/example

### PR DESCRIPTION
**Pull request by sergey-melnychuk**

### CODE REVIEW

File: tempto-core/src/test/groovy/io/trino/tempto/internal/configuration/YamlConfigurationTest.groovy [hunk 1]
```diff
@@ -60,10 +60,37 @@ list:
...
+  one:
+    q: life, universe and everything
+    a: 42
+  two:
+    - quick
+    - brown
+    - fox
+
+copy: 
+  <<: *root
```
Comment (hashhar) [inline tempto-core/src/test/groovy/io/trino/tempto/internal/configuration/YamlConfigurationTest.groovy:84]: Can we also have a test where we override keys from the anchor?

e.g. changing `one.q` to something else in under `copy`?

**Review by hashhar — COMMENTED** [review]
Looks good to me % another possible test

**Review by hashhar — APPROVED** [review]
LGTM.

@sergey-melnychuk Can you mention a justification for the version bump in the commit message so that it's possible to understand the reason in future? If there's no justification let's avoid it (in this PR at-least).

cc: @electrum PTAL if you want.

**Comment by sergey-melnychuk** [comment]
> let's avoid it

@hashhar Removing the bump, as I don't have any specific reason for it really.
I just saw PR from dependabot with bump to 1.26, but the most recent is 1.29.

**Pull request closed without merging**